contrib/docker/Makefile

```diff
@@ -65,13 +76,30 @@ endif
 
 ifeq ("$(shell echo ${OS} | grep centos)","centos74")
     RUN_AS_USER_SCRIPT=${DOCKUSER_HOME}/ngraph-test/contrib/docker/run_as_centos_user.sh
-    DOCKERFILE=Dockerfile.ngraph.centos74
+    CPU_DOCKERFILE=Dockerfile.ngraph.centos74
```

Comment:
This looks inconsistent with the "else" case below.  In the "else" case, you use conditional assigment (?=), but here you are using absolute assignment.  Should both the then and else cases use conditional, or both use absolute?